tutorials/quantum_simulation/Building 1D Rydberg Crystals.ipynb

```diff
@@ -139,16 +139,16 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    occup_list = [occupation(reg, j) for j in range(len(reg.qubits))]
</pre><p>Good catch, we should prune the tutorials of things like these</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/656/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>